api/controllers/CampaignBotController.js

```diff
@@ -366,25 +365,24 @@ class CampaignBotController {
     const dateSubmitted = Date.now();
     const submission = req.signup.draft_reportback_submission;
     submission.submitted_at = dateSubmitted;
+    const currentSignup = req.signup;
```

Comment:
This causes a bug when we call`this.renderResponseMessage(req, 'menu_completed')` at the end of the function.
When `CampaignBotController.renderResponseMessage` is passed a `'menu_completed'` parameter, it's rendering our response messages with properties we're now setting on the `currentSignup` instead of `req.signup` -- but never update the `req` when we pass it to `renderResponseMessage` .

two ideas: 
1 - Create a `const scope = req` and pass it on to `scope` to `renderResponseMessage(scope, 'menu_completed')` to make lint happy

2 -Add `// eslint-disable-line no-param-reassign` like I did in `config/router.js`. 

@sergii-tkachenko I think i'd prefer the 2nd approach because it's less changes, but do you have input on which is better practice?